lib/galaxy/util/__init__.py

```diff
@@ -77,6 +78,12 @@
     safe_relpath,
 )
 
+try:
+    shlex_join = shlex.join  # type: ignore[attr-defined]
+except AttributeError:
+    # Python < 3.8
+    shlex_join = lambda split_command: " ".join(map(shlex.quote, split_command))  # noqa: E731
```

Comment:
I forgot about E731, I am also fine with:
```suggestion
    def shlex_join(split_command):
        return " ".join(map(shlex.quote, split_command))
```